scss/mixins/_az-color-contrast.scss

```diff
@@ -30,6 +31,28 @@
     }
   }
 
+  // Change the focus-visible color if red outline should not be used with the background color.
+  @if not list.index($light-background-colors-with-red-links, $bg-color-name) {
+    a:not(.btn):not(.nav-link):not(.page-link):focus-visible,
+    button:not(.btn):not(.carousel-control-next):not(.carousel-control-prev):not(.az-gallery-grid-btn):focus-visible {
```

Comment:
We can combine the selector classes into a single :not() pseudo-class (like `a:not(.btn, .nav-link, .page-link):focus-visible`.